### PR TITLE
Fetch bundle

### PR DESCRIPTION
**Pull request by godu**
Fetch `discipline-bundle` from MOOC's API for each cards.

### CODE REVIEW

File: src/redux/middlewares/discipline-bundle.js [hunk 1]
```diff
@@ -7,24 +7,34 @@ import type {StoreState} from '../store';
...
 
 const createMiddleware = ({services}: Options): Middleware<State, Action, Dispatch<Action>> => ({
-  dispatch
+  dispatch,
+  getState
 }: MiddlewareAPI<State, Action, Dispatch<Action>>) => (
   next: Dispatch<Action>
 ): Dispatch<Action> => (action: Action) => {
   if (action.type === FETCH_REQUEST) {
+    const token = getToken(getState());
```
Comment (Fausto95) [inline src/redux/middlewares/discipline-bundle.js]: const state = getState();

File: src/layer/data/core.js [hunk 2]
```diff
@@ -98,40 +97,50 @@ export const storeDisciplineBundle = async (
...
   }
 
-  if (Object.keys(bescherelle.disciplines).includes(ref)) {
-    return Promise.resolve(bescherelle);
-  }
-
-  // Implement fetching logic here
-  return Promise.reject(new Error('API fetching not supported yet.'));
+  const response = await fetch(
+    `${host}/api/v2/disciplines/bundle?lang=${language}&conditions={"ref": ["${ref}"]}`,
```
Comment (Fausto95) [inline src/layer/data/core.js]: `cross-fetch` does not support `qs: {}` so we could
```js
 qs: {
   lang: language,
   conditions: {
     ref: [ ...]
   }
}
```

**Comment by Fausto95** [comment]
Provide some descriptions @godu 🤨

File: src/redux/middlewares/discipline-bundle.js (hunk 1 above)
Comment (VincentCATILLON) [reply src/redux/middlewares/discipline-bundle.js]: I agree, to avoid two calls to getState()

File: src/layer/data/core.js (hunk 2 above)
Comment (godu) [reply src/layer/data/core.js]: nope... https://fetch.spec.whatwg.org/#fetch-api
I'll try to find a url polyfill.

Comment (VincentCATILLON) [reply src/layer/data/core.js]: `Accept-Language` HTTP header is not enough ? https://developer.mozilla.org/fr/docs/Web/HTTP/Headers/Accept-Language

File: src/layer/data/core.js [hunk 3]
```diff
@@ -98,40 +97,50 @@ export const storeDisciplineBundle = async (
...
+      return Promise.resolve(disciplinesBundle);
+    }
+
+    if (Object.keys(onboarding.disciplines).includes(ref)) {
+      return onboarding;
+    }
+
+    if (Object.keys(bescherelle.disciplines).includes(ref)) {
+      return bescherelle;
+    }
```
Comment (VincentCATILLON) [inline src/layer/data/core.js]: These 2 courses should be removed (and __temporary__ folder too) as they will be fetched

Comment (godu) [reply src/layer/data/core.js]: Si sauf quand tu es connecté. La langue de l'utilisateur en base prend le dessus.

File: src/layer/data/core.js (hunk 2 above)
Comment (godu) [reply src/layer/data/core.js]: Yes, but user's language setting overwrite this value.

Comment (godu) [reply src/layer/data/core.js]: header => user's language => query

**Pull request merged**